packages/portfolio-contract/src/portfolio.exo.ts

```diff
@@ -409,6 +418,11 @@ export const preparePortfolioKit = (
         },
       },
       reader: {
+        getLocalAccount(): LocalAccount {
+          const { accounts } = this.state;
+          const { lca } = accounts.get('agoric') as AccountInfoFor['agoric'];
```

Comment:
I suggest a comment: we rely on the portfolio creator (openPortfolio) to add an agoric account before making the `PortfolioKit` available to any clients.